test/test_tree_2d_shallowwater.jl

```diff
@@ -0,0 +1,41 @@
+module TestExamples2DShallowWater
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+# pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
+EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_2d_dgsem")
```

Comment:
```suggestion
EXAMPLES_DIR = joinpath(examples_dir(), "tree_2d_dgsem")
```
Let's simplify this here